buildconfig/CMake/Packaging/launch_workbench.ps1.in

```diff
@@ -0,0 +1,51 @@
+################################################################
+# Note: All commands here must be COMPATIBLE with PowerShell 2.0
+################################################################
+## Setup some initial paths
+$CM_THIRD_PARTY_DIR="@THIRD_PARTY_DIR@"
+$THIRD_PARTY_DIR=$CM_THIRD_PARTY_DIR -replace '/', '\'
+$CM_SOURCE_DIR = "@PROJECT_SOURCE_DIR@"
+$SOURCE_DIR=$CM_SOURCE_DIR -replace '/', '\'
+$ERROR_REPORTER_DIR = "$SOURCE_DIR\scripts\ErrorReporter"
+
+## turn the error reporter on
+$properties = 'Mantid.properties'
+(Get-Content $properties) -replace 'usagereports\.enabled = 0', 'usagereports.enabled = 1' | Set-Content $properties
```

Comment:
Can you help me understand the need for replacing this? I'm not sure about it as it's confusing as to why a key is not being respected if you flip it yourself.

If this is the main `Mantid.properties` file then that's controlled by the Kernel/CMakeLists.txt and is defaulted on for package builds anyway. Can we avoid this?

Reply:
Hmm, so I had created this PowerShell script to be used in developer builds to conveniently launch workbench with the error reporter enabled. For ease of use I replaced the property 'usagereports', which I agree is not great practice, but thought it would be acceptable for developer builds. 

I hadn’t intended for this to replace to hand-written launch_workbench.ps1 script used in the package build (I think as it stands this script won’t be included in the package builds). Should I generalise this script for use with both the developer builds and package builds?

Reply:
Ah I see. It was in the `CMake/Packaging` directory so I assumed it was for that but it's outside the scope here to reorganise this as I see they are all in there anyway.

If this is for the developer builds then it's actually more important that we don't turn on `usagereports.enabled` as we don't want developer builds to pollute the usage statistics for everything so I think we'll just have to live with that when testing this developers will have to manually enable the reports while testing.

Also, do you think it would be easy to have 1 template `launch_workbench.ps1.in` like we do for other platforms so we don't have to keep the dev/package versions in sync?

Reply:
Ah yes of course you wouldn’t want the usagereports on for the developer builds, that was a massive oversight by me 🤦‍♂️ . I've created a common launch script which worked locally for me, but I'll admit this is the first time I've written one, so maybe could do with some improvements. While trying to replicate the previous launch script I noticed that an error was generated in the param (..), wonder if that was working before.

I'm going to start a windows package build and make sure everything builds correctly.